compiler/src/formatting/fmt.re

```diff
@@ -314,8 +314,44 @@ let build_document = (~original_source, parsed_program) => {
     | _ => failwith("Impossible: non- prefix or infix op")
     };
   }
-  and print_constant = (~loc, constant) => {
-    string(get_original_code(loc));
+  and print_constant = constant => {
+    // TODO: Add originally written values for constants to the parsetree
+    // TODO: Add locations for constants into parsetree
+    switch (constant) {
+    | PConstNumber(PConstNumberInt(value)) => string(value)
+    | PConstNumber(PConstNumberFloat(value)) => string(value)
+    | PConstNumber(PConstNumberRational(numerator, denominator)) =>
+      string(numerator) ++ string("/") ++ string(denominator)
```

Comment:
This does not handle comments. You either need full locations on the individual components to be able to capture any comments (and determine if they're before or after the slash) or you need the original code.